Functions/It.ps1

```diff
@@ -158,8 +158,12 @@ function ItImpl
 
     #mark empty Its as Pending
     #[String]::IsNullOrWhitespace is not available in .NET version used with PowerShell 2
```

Comment:
@Remco-74, exactly in this line is an answer why the sollution proposed by you doesn't work with PowerShell 2.

Can you change your code to use the old detection method when it's run with PowerShell 2.0 and the new one for the newer version of PowerShell?